conan/test_package/CMakeLists.txt

```diff
@@ -1,13 +1,44 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/cmake)
-
 find_package(userver REQUIRED)
```

Comment:
Please add usages of `GrpcTargets.cmake` (see `samples/grpc_service`) and `UserverTestsuite.cmake` (see the service template) to prove and demonstrate how they are usable